build/dependencies.props

```diff
@@ -33,6 +33,15 @@
   <ItemGroup>
     <ExternalDependency Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0-preview2-25711-01" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="Microsoft.Extensions.DependencyModel" Version="2.1.0-preview2-25711-01" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="Microsoft.NETCore.App" Version="2.1.0-preview1-25907-02" TargetFramework="netcoreapp2.1" Source="$(DotNetCoreFeed)" Mirror="true">
+      <!-- Multiple versions of this package required to support all netcoreapp versions -->
+      <NoWarn>KRB2004</NoWarn>
+      <VariableName>MicrosoftNETCoreApp21PackageVersion</VariableName>
+    </ExternalDependency>
+    <ExternalDependency Include="Microsoft.NETCore.DotNetAppHost" Version="2.1.0-preview1-25907-02" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="Microsoft.NETCore.DotNetHostPolicy" Version="2.1.0-preview1-25907-02" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="Microsoft.NETCore.DotNetHostResolver" Version="2.1.0-preview1-25907-02" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="Microsoft.NETCore.Platforms" Version="2.1.0-preview1-25831-01" Source="$(DotNetCoreFeed)" Mirror="true" />
```

Comment:
Why does `Microsoft.NETCore.Platforms` have a different version than the rest?

Reply:
See the dependencies node here: https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App